SDPi_Supplement/asciidoc/volume0/tf0-ch-9-copyrights.adoc

```diff
@@ -1,4 +1,14 @@
-= TF-0 Chapter 9: Copyrights
+
+// = TF-0 Chapter 9: Copyrights
+
+NOTE:  Migrate content from the Word document here
+
+// 9
+
+:sectnums:
```

Comment:
Please remove. `:sectnums:` does not have no functionality anymore as it is overwritten by our custom scripts.

Reply:
OK ... agreed ... but where do I edit this out?  In my local copy and then push for the PR to include OR somewhere else now that we are in the review of the PR?

NOTE:  First time through this part of the process ... so "new" ... for this newbie !

Reply:
Line removed